generators/typescript/sdk/client-class-generator/src/request-parameter/RequestBodyParameter.ts

```diff
@@ -42,6 +42,10 @@ export class RequestBodyParameter extends AbstractRequestParameter {
         throw new Error("Cannot reference query parameter because request is not wrapped");
     }
 
+    public getReferenceToPathParameter(): ts.Expression {
+        throw new Error("Cannot reference path parameter because request is not wrapped");
+    }
```

Comment:
Does this need to be conditional? If inlinePathParameters is false, `getReferenceToPathParameter` should just return the name of the positional path parameter argument, right? Or is that not actually relevant here?

Reply:
This class is already in the context of a body request parameter, so if the body request parameter is not a wrapper, there's no path parameters on the body request.